Changes

```diff
@@ -471,6 +471,9 @@ OCaml 5.0
   Scherer, Thomas Leonard, Tom Ridge, Xavier Leroy, Luc Maranget, Fabrice
   Buoro, Olivier Nicole, Guillaume Munch-Maccagnoni, Jacques-Henri Jourdan)
 
+ -#11640: Added Flambda commonly-used options to the ocamlopt.1 file in the man folder
+  (Nangah Amandine, review by David Allsopp, Florian Angeletti, Sébastien Hinderer, Vincent Laviron)
+
```

Comment:
There is a constraint of at most 80 characters by line in this file. Moreover, it is better to describe the change as visible to users:
```
 - #11640: Add Flambda commonly-used options to the ocamlopt manpage
  (Nangah Amandine, review by David Allsopp, Florian Angeletti,
   Sébastien Hinderer, and Vincent Laviron)
```